app/analysis/index/controller.js

```diff
@@ -162,6 +162,8 @@ const AnalysisController = Controller.extend(AnalysisQueryParams.Mixin, {
 
   @oneWay('session.data.authenticated.token') jwt: null,
 
+  _scrollOffset: 0,
```

Comment:
Reset this property to zero on reset (line 191)

Reply:
Done